woocommerce-services.php

```diff
@@ -266,7 +266,7 @@ public static function plugin_deactivation() {
 			require_once __DIR__ . '/classes/class-wc-connect-wcst-to-wcshipping-migration-state-enum.php';
 
 			$migration_state = WC_Connect_Options::get_option( 'wcshipping_migration_state' );
-			if ( $migration_state === WC_Connect_WCST_To_WCShipping_Migration_State_Enum::STARTED ) {
+			if ( $migration_state !== WC_Connect_WCST_To_WCShipping_Migration_State_Enum::COMPLETED ) {
```

Comment:
When the migration deactivates WCS&T, we won't be able to call the `connect/migration-flag` endpoint because the RESTful controller `WC_REST_Connect_Migration_Flag_Controller` is also deactivated. As a result, we rely on the `plugin_deactivation` hook to update the migration process.

If our migration state has started but not completed, and WCS&T is deactivated, then this means that the migration has successfully moved to the last step. We can safely update the migration flag to "completed".